src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+      </div>
```

Comment:
Using multiple static elements like this indicates that you might be manually adding elements for each day. Consider using a more dynamic approach, such as generating these elements with JavaScript based on the month's length and the starting day. This will make your code more maintainable and scalable.